ambry-protocol/src/main/java/com.github.ambry.protocol/PutRequest.java

```diff
@@ -388,6 +381,7 @@ public String toString() {
       sb.append(", ").append("blobType=").append(blobType);
       sb.append(", ").append("blobSize=").append(blobSize);
       sb.append(", ").append("crc=").append(receivedCrc);
+      sb.append(", ").append("BlobKeyAvailable=").append(blobKey != null);
```

Comment:
better to not log the keys themselves for security reasons. You could log whether a key is present or not.

Reply:
which is what you are doing - ignore my comment.